### ISSUE TITLE
Implement 'Custom' Mouse Pointer Style

### ISSUE THREAD

**Issue by justingarfield** [justingarfield/dotfiles#1]
Right now this repository only supports `White`, `Black`, and `Inverted` options for **Accessibility** -> **Mouse pointer and touch** -> **Mouse Pointer Style**.

If the need arises, may want to implement `Custom` option as well. This is a bit more involved though, as when you choose `Custom`, it not only sets the Registry Keys/Values below, but also generates a separate copy of every Cursor file with the selected color to the user's `%LOCALAPPDATA%\Microsoft\Windows\Cursors` folder.

```powershell
Custom = [MousePointerStyle]::new(
    "Windows Aero",                         # Name
    "$appDataLocalCursors\busy_eoa.cur",    # AppStarting
    "$appDataLocalCursors\arrow_eoa.cur",   # Arrow
    "$appDataLocalCursors\cross_eoa.cur",   # Crosshair
    6,                                      # CursorType
    "$appDataLocalCursors\link_eoa.cur",    # Hand
    "$appDataLocalCursors\helpsel_eoa.cur", # Help
    "$appDataLocalCursors\ibeam_eoa.cur",   # IBeam
    "$appDataLocalCursors\unavail_eoa.cur", # No
    "$appDataLocalCursors\pen_eoa.cur",     # NWPen
    2,                                      # SchemeSource
    "$appDataLocalCursors\move_eoa.cur",    # SizeAll
    "$appDataLocalCursors\nesw_eoa.cur",    # SizeNESW
    "$appDataLocalCursors\ns_eoa.cur",      # SizeNS
    "$appDataLocalCursors\nwse_eoa.cur",    # SizeNWSE
    "$appDataLocalCursors\ew_eoa.cur",      # SizeWE
    "$appDataLocalCursors\up_eoa.cur",      # UpArrow
    "$appDataLocalCursors\wait_eoa.cur"     # Wait
)
```

**Comment by justingarfield** [comment]
Need to implement this for **Mouse Pointer Size**, as adjusting that causes usage of `Custom` anyway.
